code/variables.tf

```diff
@@ -424,3 +424,9 @@ variable "io_apim_productid" {
   type    = string
   default = null
 }
+
+# Recaptcha
+variable "recaptcha_secret_key" {
```

Comment:
@AleDore who is going to provide me this secret key? Should it be different from production and uat?

Reply:
Nope, we can use the same used in io-pay